config/clientConfig.js

```diff
@@ -0,0 +1,40 @@
+const serverPort = process.env.PORT || 3122
+
+const completeConfig = {
+
+  default: {
+    appName: process.env.APPNAME || 'Voluntarily NZ',
+    serverPort,
+    env: process.env.NODE_ENV || 'development',
+    jsonOptions: {
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json'
+      }
+    },
+    auth: {
+      AUTH0_CLIENT_ID: 'S4yd4VgZ92NIjhwO3vt4h0Gifb9mXv1k',
+      AUTH0_CLIENT_DOMAIN: 'voluntarily.au.auth0.com'
+    },
+    apiVersion: 'v1',
+    BADGR_API: process.env.BADGR_API || 'https://api.au.badgr.io'
```

Comment:
Does the client need access to the Badgr API? Aren't all requests proxied through our own Badge API?

Reply:
correct - will fix.